src/viperleed/utilities/poscar/modify_vacuum.py

```diff
@@ -12,31 +12,52 @@
 __license__ = 'GPLv3+'
 
 from copy import deepcopy
+from dataclasses import dataclass
 import logging
 
 import numpy as np
 
-from viperleed.calc.classes.slab.errors import (NotEnoughVacuumError,
+from viperleed.calc.classes.slab.errors import (
+                                                NotEnoughVacuumError,
+                                                VacuumError,
                                                 WrongVacuumPositionError)
```

Comment:
```suggestion
from viperleed.calc.classes.slab.errors import NotEnoughVacuumError
from viperleed.calc.classes.slab.errors import VacuumError
from viperleed.calc.classes.slab.errors import WrongVacuumPositionError
```

Better keep one explicit import per line. Makes it easier to add/remove imports later, as (i) the commit involves only a single line, and (ii) you don't risk forgetting a trailing comma.

Reply:
Fine by me, but I think that goes against the pylint suggestions that you are usually so strict on.

Reply:
I'm not sure I understand which suggestion does this go against:
- [ungrouped-imports](https://pylint.readthedocs.io/en/stable/user_guide/messages/convention/ungrouped-imports.html) only says one should have all `import` statements from the same module in consecutive lines
- [PEP-8](https://peps.python.org/pep-0008/#imports) does not seem to state anything about deciding to have one import per line.

Could you please share a reference to where this is stated to be an anti-pattern?

Reply:
As far as I understand, PEP328: https://peps.python.org/pep-0328/

Reply:
That looks like the PR that made parantheses possible in imports, which I think are still acceptable when one of the import statements goes beyond the 79 char limit. In that case, we have adopted the following style:

```python
from long_module_name.with.a_lot.of_submodules import (
    some_short_name,
    some_very_long_name_that_would_not_fit_pep8,
    )
```

Notice that, like in the case of this comment, there is one imported quantity per line.